src/Raven.Server/Web/Studio/StudioTasksHandler.cs

```diff
@@ -16,7 +16,7 @@ namespace Raven.Server.Web.Studio
     public class StudioTasksHandler : RequestHandler
     {
         //TODO: split to specific handler
-        [RavenAction("/databases/*/studio-tasks/config", "GET")]
+        [RavenAction("/databases/*/studio-tasks/config", "GET", AuthorizationStatus.ServerAdmin)]
```

Comment:
@ml054 Can we drop this?
At any rate, this looks like something that any user should be able to get, no?